cmd/gctcli/commands.go

```diff
@@ -1748,12 +1743,6 @@ func cancelOrder(c *cli.Context) error {
 		return errInvalidAsset
 	}
 
-	if c.IsSet("wallet_address") {
-		walletAddress = c.String("wallet_address")
-	} else {
-		walletAddress = c.Args().Get(5)
-	}
-
 	if c.IsSet("side") {
 		orderSide = c.String("side")
 	} else {
```

Comment:
`orderSide = c.Args().Get(6)` might need to be changed to `orderSide = c.Args().Get(5)`

Reply:
Oh, this so doesn't feel right 😂 
Not going to get distracted, but yoinking positional args like this feels like we're playing with fire.
Stepping away, just touching it the way it wants:

Fixed https://github.com/gbjk/gocryptotrader/commit/877ef6ecfc